cmd/dockerd/daemon.go

```diff
@@ -244,6 +254,30 @@ func (cli *DaemonCli) start(opts daemonOptions) (err error) {
 		logrus.Fatalf("Error starting cluster component: %v", err)
 	}
 
+	builderStateDir := filepath.Join(cli.Config.Root, "builder")
+
+	fsCache, err := fscache.NewFSCache(fscache.Opt{
+		Backend: fscache.NewNaiveCacheBackend(builderStateDir),
+		Root:    builderStateDir,
+		GCPolicy: fscache.GCPolicy{ // TODO: expose this in config
```

Comment:
Should'nt this policy be configurable prior to merge the first version to master ?